klasses/utils.py

```diff
@@ -150,3 +155,26 @@ def unrevoke_certificate(user, bootcamp_run):
         result["msg"] = "Valid user and bootcamp_run must be provided."
 
     return result
+
+
+def manage_user_certificate_blocking(users, block_state):
+    """Block users for getting certificates in all bootcamp run enrollments"""
+    result = {"updated": False}
+    if users:
+        BootcampRunEnrollment.objects.filter(user__email__in=users, active=True).update(
```

Comment:
```suggestion
        BootcampRunEnrollment.objects.filter(user__email__in=users).update(
```
might not create much impact but I guess we shouldn't leave any loophole.